pyproject.toml

```diff
@@ -52,5 +52,5 @@ tox = "^3.19"
 target-version = ['py36', 'py37', 'py38']
 
 [build-system]
-requires = ["poetry>=1,<2"]
+requires = ["poetry>=1,<2", "setuptools"]
```

Comment:
This was added to make `pip install -e .` work; I need to research whether `poetry` has something built in. Not really for landing.